rest/route/host.go

```diff
@@ -621,7 +621,9 @@ func (h *hostIpAddressGetHandler) Run(ctx context.Context) gimlet.Responder {
 	if host == nil {
 		return gimlet.MakeJSONErrorResponder(gimlet.ErrorResponse{
 			StatusCode: http.StatusNotFound,
-			Message:    fmt.Sprintf("host with ip address '%s' not found", h.IP),
+			Message: fmt.Sprintf("host with ip address '%s' may not exist "+
```

Comment:
This is a good start in terms of help text, however really the purpose of this command is "I know this IP address is causing an issue but I don't know who's it is." It may be easier to just say "may not exist or may not have IP address stored."

Reply:
message simplified

Reply:
The reason I though it may be helpful to tell whoever is using the command that the ip address will be stored after a restart is so that if they use it and get frustrated that the IP address is not stored, they view it as less of an error and more of a "let me tell person x to restart their machine so it's their for future use". 
But I hear the argument for simplicity.